WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
```

Comment:
Used label_tag...hope it's the correct procedure.
The buttons now are not aligned :P